artificer/src/test/java/net/technolords/tools/artificer/input/ArtifactResourceVisitorTest.java

```diff
@@ -14,16 +23,64 @@
     /**
      * Test case #1: Test classification of a resource.
      */
-    @Test
-    public void testClassificationOfResource() {
-        LOGGER.info("Todo: implementation of test case");
+    @DataProvider(name = "dataSetClassifyResource")
+    public Object[][] dataSetClassifyResource() {
+        return new Object[][] {
+                {"ArtificerImplTest.class","D:/commit2Learn/techno-tools/artificer/target/test-classes/net/technolords/tools/artificer/"},
+                {"abc","D:/commit2Learn/techno-tools/artificer/src/test/resources/data/"}
```

Comment:
Path's are absolute while they should be relative. This is failing on my system.